applications/asset_tracker_v2/doc/location_module.rst

```diff
@@ -41,8 +41,7 @@ The location module receives configuration updates as payload in the following t
 
 The module sends the following events based on the outcome of the location request:
 * ``LOCATION_MODULE_EVT_GNSS_DATA_READY``: GNSS position has been acquired
-* ``LOCATION_MODULE_EVT_NEIGHBOR_CELLS_DATA_READY``: neighbor cell measurements have been obtained
-* ``LOCATION_MODULE_EVT_WIFI_ACCESS_POINTS_DATA_READY``: Wi-Fi access points have been obtained
+* ``LOCATION_MODULE_EVT_CLOUD_LOCATION_DATA_READY``: neighbor cell measurements and/or Wi-Fi access points have been obtained
```

Comment:
```suggestion
* ``LOCATION_MODULE_EVT_CLOUD_LOCATION_DATA_READY``: neighbor cell measurements or Wi-Fi access points (or both) have been obtained
```
We always try to avoid and/or (although it is very convenient)